packages/app/src/pages/Debug.vue

```diff
@@ -55,7 +55,14 @@ const shouldPauseQuery = computed(() => {
 const query = useQuery({ query: DebugDocument, variables, pause: shouldPauseQuery, requestPolicy: 'network-only' })
 
 const isLoading = computed(() => {
-  return !relevantRuns.value || query.fetching.value
+  const relevantRunsHaveNotLoaded = !relevantRuns.value
```

Comment:
I wonder if we are doing load states wrong in general, we have a few of these big conditionals in the app. Not looking for you to change anything right now -- just curious, did you have any instinct around how we might be better able to handle load states in general in the future (it's kind of the core problem with the flickering - our lack of clarify around how/when data loads in our app).

Reply:
I have a much better understanding after this project on how the Graphql is getting resolved between local data and Cloud data.  The backend is doing a few things for the sake of a quick response that causes other data to come back later and create reactive responses later on.  I am going to write up a guide for this and will seek out some feedback on how this could be improved.  

The complexity here is because there is not a good way to run a Cloud query that is dependent on the result of another Cloud query without coming all the way back to the front end.  This is due to the way the local and Cloud schema are "stiched" together. More to come!